compliance-by-design-demo/server/app.js

```diff
@@ -0,0 +1,46 @@
+const fetch = require("node-fetch")
```

Comment:
I prefer import statements over require. This also keeps it consistent with the react code.

Reply:
There is a difference between `require` and `import`.
Require is used to import modules automatically from `node_modules`, while import is used to import files. 
I think it's fine like this.

Reply:
That's not accurate. `import` works fine with node_modules, and it allows for deconstructed imports. Again, I'd prefer to keep js styles the same for all js code in this project. 

Why don't we let @pimotte be the final arbiter on this. This has been an ICTU project that we only just joined after all.

Reply:
`import` in node.js requires the `--experimental-modules` flag or babel, at least up to node v12, which is latest LTS, so require seems fine to me in this context. (In the frontend, using `import` is strongly preferred)

Reply:
If it doesn't work out of the box (I guess I've only worked on typescript or babel based project). Let's just stick to `require` in the backend then. Fine by me 👍